src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ CHANGELOG
  * Deprecated `Symfony\Component\Form\Extension\Validator\Util\ServerParams` in favor of its parent class `Symfony\Component\Form\Util\ServerParams`
  * Added the `html5` option to the `ColorType` to validate the input
  * Deprecated `NumberToLocalizedStringTransformer::ROUND_*` constants, use `\NumberFormatter::ROUND_*` instead
+ * Added a `html5` option to `MoneyType` and `PercentType`, to use `<input type="number" />`
```

Comment:
Since 5.1 is in beta now this needs to be added to a new `5.2.0` section.

Reply:
Indeed :S 
I'm updating, thank you for your review.